Solves the issue reported in Problem Report bin/102701.

Can't delete an inet6 alias address from an interface.

Index: fbsd-6.1/sbin/ifconfig/af_inet6.c
===================================================================
--- fbsd-6.1.orig/sbin/ifconfig/af_inet6.c
+++ fbsd-6.1/sbin/ifconfig/af_inet6.c
@@ -515,7 +515,7 @@ static struct afswtch af_inet6 = {
 	.af_settunnel	= in6_set_tunnel,
 	.af_difaddr	= SIOCDIFADDR_IN6,
 	.af_aifaddr	= SIOCAIFADDR_IN6,
-	.af_ridreq	= &in6_addreq,
+	.af_ridreq	= &in6_ridreq,
 	.af_addreq	= &in6_addreq,
 };
 
